project-management.qmd

```diff
@@ -28,17 +28,20 @@ Our group uses the following hierarchy of meetings, from brief daily status repo
 
 ## Time Tracking
 
-Every individual in the group contributes to multiple projects.
+We use [Toggl](https://toggl.com) to track the time each group member spends working on each of the (many) projects that we all contribute to.
```

Comment:
```suggestion
We use [Toggl](https://track.toggl.com) to track the time each group member spends working on each of the (many) projects that we all contribute to.
```
Toggl make another product in addition to the time tracker, this URL goes directly to the tracker